SharedPackages/VPN/Sources/VPN/Subscription/VPNSubscriptionNotificationPixel.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  VPNSubscriptionNotificationPixel.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import PixelKit
+import Subscription
+
+public enum VPNSubscriptionNotificationPixel: PixelKitEventV3 {
+    case vpnEnabled(isSubscriptionActive: Bool?,
```

Comment:
I was going to use a string here, but having a parameter named `isSubscriptionActive` with a string value looked extremely weird.